lib/redis_memo/memoize_query/invalidation.rb

```diff
@@ -258,9 +253,12 @@ def self.select_by_new_ids(model_class, target_id)
     end
   end
 
-  def self.bulk_operations_invalidation_limit
-    ENV['REDIS_MEMO_BULK_OPERATIONS_INVALIDATION_LIMIT']&.to_i ||
-      RedisMemo::DefaultOptions.bulk_operations_invalidation_limit ||
-      10000
+  def self.select_by_conflict_target_relation(model_class, relation)
+    RedisMemo::Tracer.trace(
+      'redis_memo.memoize_query.invalidation',
+      "#{__method__}##{model_class.name}",
+    ) do
+      RedisMemo.without_memo { relation.reload }
```

Comment:
Just curious, what's the motivation behind choosing relation.reload, relation.to_a (as is done in select_by_new_ids method), vs just relation.load?

Reply:
We use the same relation twice, query it once before updating, and another time after updating